cpplib/Tests/MainTest.cpp

```
@@ -93,6 +97,204 @@ struct FMIC_TS {
+	Diagram diag(buildresult.atoms.points, bonds, cell, bools);
+	auto ce = diag.extractCells();
+
+	cpplib::voronoi::VoronoiFused vf(ce, cell.fracToCart());
+
+	auto critical_points = cpplib::BaderOperator::GenerateInitialCriticalPoints(vf, cell.fracToCart());
+
+	// Expand Point Net
+	// 1. Calculate theoretical radius.
+
+	const auto ED_cutoff = cpplib::BaderOperator::GetOptimalRadius(3.0, 1.0E-6, ElectronDensitySplines, buildresult.atoms.types);
+	const size_t max_type = cpplib::ElectronDensitySplines.size()-1;
```

Clang (14.0.6)

[warning] variable 'max_type' is not initialized

(cppcoreguidelines-init-variables)